test/FunctionalTests/Taupo/Source/Taupo.Astoria/Client/DataServiceContextHttpTracker.cs

```diff
@@ -420,7 +421,7 @@ private TReturn CleanupStateOnFailure<TReturn>(Func<TReturn> action)
                 }
             }
 
-            private class TestHttpWebRequestMessage : HttpWebRequestMessage
+            private class TestHttpWebRequestMessage : HttpClientRequestMessage

```

Comment:
I wish we retained the `HttpWebRequestMessage` tests, and added `HttpClientRequestMessage` tests.
- Would have allowed us to capture any regressions